docs/tutorials/getting_started.ipynb

```diff
@@ -189,6 +189,7 @@
    "outputs": [],
    "source": [
     "chesapeake_root = os.path.join(tempfile.gettempdir(), \"chesapeake\")\n",
+    "os.makedirs(chesapeake_root, exist_ok=True)\n",
```

Comment:
This is actually a bug in TorchGeo: #1605. Pretty minor so I don't think it's worth holding up the release for.